src/Storefront/Theme/Aggregate/ThemeTranslationEntity.php

```diff
@@ -32,23 +43,51 @@ public function setDescription(?string $description): void
         $this->description = $description;
     }
 
+    /**
+     * @deprecated tag:v6.8.0 - Will be removed. Use label snippet keys from structured fields instead
+     *
+     * @return array<string, string>|null
+     */
     public function getLabels(): ?array
     {
+        Feature::triggerDeprecationOrThrow('v6.8.0.0', Feature::deprecatedClassMessage(self::class, 'v6.8.0.0'));
```

Comment:
The class won't be deprecated, right? Also check the other PHP deprecations
```suggestion
        Feature::triggerDeprecationOrThrow('v6.8.0.0', Feature::deprecatedMethodMessage(self::class,'getLabels' , 'v6.8.0.0'));
```

Reply:
Ah, okay. Didn't know it could get more specific. Nice catch!